test/library/randomized_benchmarking/test_randomized_benchmarking.py

```diff
@@ -588,13 +588,21 @@ def test_correct_1q_depolarization(self):
         result_q1 = analysis_1qrb_q1.run(self.expdata_1qrb_q1, replace_results=False)
         self.assertExperimentDone(result_q1)
 
+        analysis_2qrb = rb.RBAnalysis()
+        analysis_2qrb.set_options(
+            outcome="00",
+        )
+        result_2qrb = analysis_2qrb.run(self.expdata_2qrb)
+        result_2qrb.block_for_results()
+        cx_epg_raw = result_2qrb.analysis_results("EPG_cx")
+
         analysis_2qrb = rb.RBAnalysis()
         analysis_2qrb.set_options(
             outcome="00",
             epg_1_qubit=result_q0.analysis_results() + result_q1.analysis_results(),
         )
         result_2qrb = analysis_2qrb.run(self.expdata_2qrb)
-        self.assertExperimentDone(result_2qrb)
+        result_2qrb.block_for_results()
```

Comment:
Why did you replace `assertExperimentDone` by `block_for_results`?
Same below.

Reply:
No strong reason. I'll follow the convention.

Reply:
Updated to use `assertExperimentDone` in [15c2782](https://github.com/Qiskit/qiskit-experiments/pull/898/commits/15c27822bc93be0a80a3d3962f06261cf4d952e2)